ControlsKit/CTKPlaceholderTextView.m

```diff
@@ -84,12 +84,23 @@ - (void)setText:(NSString *)text {
   [self updatePlaceholderVisibilityFromNotification:nil];
 }
 
+- (void)setAttributedText:(NSAttributedString *)attributedText {
+	[super setAttributedText:attributedText];
+
+	[self updatePlaceholderVisibilityFromNotification:nil];
+}
+
 - (void)layoutSubviews {
   [super layoutSubviews];
 
   [self updatePlaceholderFrame];
 }
 
+- (void)setAdjustsFontForContentSizeCategory:(BOOL)adjustsFontForContentSizeCategory {
+	[super setAdjustsFontForContentSizeCategory:adjustsFontForContentSizeCategory];
+	self.placeholderLabel.adjustsFontForContentSizeCategory = adjustsFontForContentSizeCategory;
+}
```

Comment:
Should you have a line break between the two lines, for consistency with the `setAttributedText:` function?